tensorflow/lite/micro/kernels/detection_postprocess_test.cc

```diff
@@ -460,4 +460,57 @@ TF_LITE_MICRO_TEST(
       /* input3 min/max */ 0.0, 100.5);
 }
 
+TF_LITE_MICRO_TEST(
+    DetectionPostprocessQuantizedSwithNoBackgroundClassAndKeypointsStableSort) {
+  int kInputShape1[] = {3, 1, 6, 5};
+  int kInputShape2[] = {3, 1, 6, 2};
+
+  // six boxes in center-size encoding
+  const float kInputData1[] = {
+      0.0, 0.0,  0.0, 0.0, 1.0,  // box #1
+      0.0, 1.0,  0.0, 0.0, 1.0,  // box #2
+      0.0, -1.0, 0.0, 0.0, 1.0,  // box #3
+      0.0, 0.0,  0.0, 0.0, 1.0,  // box #4
+      0.0, 1.0,  0.0, 0.0, 1.0,  // box #5
+      0.0, 0.0,  0.0, 0.0, 1.0,  // box #6
+  };
+
+  // class scores - two classes without background
+  const float kInputData2[] = {0.015625, 0.007812, 0.003906, 0.015625,
+                               0.015625, 0.007812, 0.019531, 0.019531,
+                               0.007812, 0.003906, 0.003906, 0.003906};
+
+  constexpr int kInputElements1 =
+      1 * 6 * 5;  // kInputShape1[1] * kInputShape1[2] * kInputShape1[3];
```

Comment:
nit: I think that if you make kInputShape a constexpr, then you could initialize kInputElements1 with kInpuShape

Reply:
done, I realized the uint8 unit tests had been removed

Reply:
uint8 removal for detection_postprocess was likely an oversight on our part. Can you confirm that the CMSIS-NN team needs continued support for this? https://github.com/tensorflow/tflite-micro/issues/457